src/main/functions/DidJWT.ts

```diff
@@ -273,3 +273,12 @@ export function toSIOPRegistrationDidMethod(didOrMethod: string) {
   const split = prefix.split(':');
   return `${split[0]}:${split[1]}:`;
 }
+
+export function toQualifiedDidMethod(didOrMethod: string) {
```

Comment:
What does qualified mean? Because the end result is not a DID method according to DID-core. That is why there is the toSiopRegistrationDidMethod. That method needs this body basically (so removing the last : )

Reply:
removed this method and changed the impl of `toSiopRegistrationDidMethod` to return `did:example`